src/js/jsx/shared/TitleHeader.jsx

```diff
@@ -38,6 +39,12 @@ define(function (require, exports, module) {
         render: function () {
             var workingTitle = this.props.title;
 
+            var classNameSet = {
+                "section-title": true
+            };
+
+            var className = classnames(classNameSet, this.props.className);
```

Comment:
this is a nit, but to echo @shaoshing's suggestion from the first PR, this would be better as: `var className = classnames("section-title", this.props.className);` and remove the three previous lines.